autogen/oai/client.py

```diff
@@ -1,31 +1,41 @@
 from __future__ import annotations
 
+import inspect
```

Comment:
You have enabled `isort`, right? @sonichi , @qingyun-wu , this reminds me of https://github.com/microsoft/FLAML/pull/1259 , I think it's a good idea to enable isort here as well.

Reply:
Yes, that's correct.